editor/editor_run_native.cpp

```diff
@@ -66,6 +66,32 @@ void EditorRunNative::_notification(int p_what) {
 	}
 
 	if (p_what == NOTIFICATION_PROCESS) {
+		for (int i = 0; i < EditorExport::get_singleton()->get_export_platform_count(); i++) {
+			Ref<EditorExportPlatform> eep = EditorExport::get_singleton()->get_export_platform(i);
+			Ref<EditorExportPreset> preset;
+
+			for (int j = 0; j < EditorExport::get_singleton()->get_export_preset_count(); j++) {
+				Ref<EditorExportPreset> ep = EditorExport::get_singleton()->get_export_preset(j);
+				if (ep->is_runnable() && ep->get_platform() == eep) {
+					preset = ep;
+					break;
+				}
+			}
+
+			if (preset.is_null()) {
+				menus[i]->set_disabled(true);
```

Comment:
This will crash if `menus[i]` is null (which might be possible, considering the code in ENTER_TREE). You need to add a null check.